packages/intents-sdk/src/bridges/poa-bridge/poa-bridge.ts

```
@@ -175,10 +175,16 @@ export class PoaBridge implements Bridge {
 				logger: args.logger,
 			},
 		);
-
+		const poaBridgeFee = BigInt(estimation.withdrawalFee);
```

Can we actually have a 0 fee in POA bridge and maybe we should throw an exception in case the fee is actually 0 ?